### PR TITLE
Update readme with TOC, fixes #1

### LINKED ISSUE #1: Add table of contents to README

### PR DESCRIPTION
**Pull request by elsonlim**
Fixes #1

@tulikavijay please take a look! 😄 

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,17 @@ are now using  the  Web  to  access  vast  amounts  of  information  and  resour
 
 Aim of this Project is to develop a website that assists students and teachers alike to discover online courses, to enroll in courses from different sources, to compare these sources and to track their progress. Students can decide on their own pace and with which session they would like to begin their studies.
 
+# Table of Contents
+
+- [Hacking Online Education](#hacking-online-education)
```
Comment (tulikavijay) [inline README.md]: I think this is not needed.

File: README.md [hunk 2]
```diff
@@ -4,6 +4,17 @@ are now using  the  Web  to  access  vast  amounts  of  information  and  resour
...
+- [Table of Contents](#table-of-contents)
```
Comment (tulikavijay) [inline README.md]: This also is not required in toc.

Comment (tulikavijay) [reply README.md]: line 10 i mean

**Comment by tulikavijay** [comment]
Thanks for the PR! only slight changes and i'll merge it shortly.

File: README.md (hunk 1 above)
Comment (tulikavijay) [reply README.md]: Maybe take a look at [this](https://github.com/tulikavijay/CBT-therapy#helping-women-cope-with-depression-and-anxiety)

**Comment by elsonlim** [comment]
@tulikavijay made the changes you requested.

Looks like that now:

image

😄

**Pull request merged**